### ISSUE TITLE
Links zu externen Seiten kennzeichnen

### ISSUE THREAD

**Issue by Koala** [redaxo/docs#259]
Anstatt Links als "extern" zu kennzeichen (im Menü) oder gar nicht (im normalen Text) möchte ich anregen, Links die zu externen Seiten führen, durch einen kleinen Pfeil oder ähnlichem zu kennzeichnen. Eventuell wäre das sogar Redaxo/FOR einheitlich machbar?
Diskussionen dazu gern auch über Slack.

Zum Beispiel ganz simpel so:

```
<a href="https://friendsofredaxo.github.io/tricks/">REDAXO-Tricks &#x2197; </a>
```

![grafik](https://user-images.githubusercontent.com/330686/216675006-cf584b5c-c250-4720-ac86-bbd80a39cbf6.png)

Hier findet sich eine schöne Übersicht:
https://www.w3schools.com/charsets/ref_utf_arrows.asp

Wie wird das heutzutage mit externen Links gehandhabt? Beim klick darauf heist es "Huch, bin im selben Browser-Tab aber auf einer anderen Seite und finde nicht mehr zur ursprünglichen Seite zurück" oder gleich einen neuen Tab öffnen lassen?

**Comment by alxndr-w** [comment]
> Wie wird das heutzutage mit externen Links gehandhabt?

Best practice ist wohl, es dem Nutzer zu überlassen, häufige Praxis ist es zu forcieren.

**Comment by eaCe** [comment]
https://css-tricks.com/use-target_blank/

**Comment by Koala** [comment]
Hier gibt es auch eine gute Variante, die für alle externen Links taugt, ohne jeden einzelnen Link von Hand anpassen zu müssen. 

https://css-irl.info/styling-external-links-with-attribute-selectors/

**Comment by alxndr-w** [comment]
@Koala hast du denn den @eaCe verlinkten Artikel gelesen?

**Comment by eaCe** [comment]
@alxndr-w ich denke, dort geht es nur um das Kennzeichnen eines externen Links, das bezieht sich nicht auch _blank.

**Comment by Koala** [comment]
Ja, habe ich gesehen. Das bezieht sich aber auf den zweite Teil (die Frage) meines Beitrages. 
Mir geht es hauptsächlich um das kenntlich machen eines (externen) Links, um auf den ersten Blick erkennen zu können, ob ich von der Seite weggeführt werde oder nicht, ohne den Fokus zum Inhalt zu verlieren.
Auf Systemen, mit denen die Zieladresse eines Links durch überfahren mit einem Mauskursor irgendwo im Browser (meist links unten) ersichtlich ist, kann ein externer Link zwar erkannt werden, der Fokus auf den Inhalt geht aber verloren.

Bei Mauszeigerlosen (Touch) Systemen ist der Benutzer dem Überraschungseffekt völlig ausgesetzt, ohne die Wahl zu haben, ob er die Seite im selben Browsertab verlassen möchte, oder den Link in einem neuen Tab öffnen möchte.
